rkt/run.go

```diff
@@ -92,6 +94,11 @@ func runRun(args []string) (exit int) {
 		return 1
 	}
 
+	if len(flagPodManifest) > 0 && (len(flagVolumes) > 0 || len(flagPorts) > 0 || flagInheritEnv || !flagExplicitEnv.IsEmpty() || rktApps.Count() > 0 || flagLocal) {
```

Comment:
Because it assumes the ACI is already in the store, so `flagLocal` is unnecessary.
I just eliminated all flags unnecessary or conflicted.

Sure I can move it to a func... But since this one is slightly different with the one in prepare.go, so we will have two functions with just one if clause...